src/test/java/seedu/address/logic/commands/AddVisitCommandTest.java

```diff
@@ -21,7 +27,7 @@
 public class AddVisitCommandTest {
     @Test
     public void constructor_nullLocation_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> new AddVisitCommand(null));
+        assertThrows(NullPointerException.class, () -> new AddVisitCommand(null, null, null));
```

Comment:
Now that there are three parameters for the constructors, perhaps it could be better if each parameter is tested for NullPointerException in a separate method.

Reply:
Have added additional assertions.